1.0/resources/tools.md

```diff
@@ -0,0 +1,222 @@
+
+[polyserve](https://github.com/PolymerLabs/polyserve) is simple web server for serving `bower_components`, locally. It's useful when developing your own elements. 
+
+polyserve serves the component from the current directory as `/components/{element-name}/`, where `element-name` is defined as the name in `bower.json`. All other dependencies are served from `./bower_components/`.
+
+Install:
+
+    npm install -g polyserve
+
+Usage: 
+
+    cd my-element/
+    polyserve -p 8080
+
+Source: [github.com/PolymerLabs/polyserve](https://github.com/PolymerLabs/polyserve)
+
+### <b>polygit</b> - CDN web service for serving components {#polygit}
+
+[polygit](http://polygit.org/) is proxy server for serving components from a CDN. **It is not meant to be used for production apps**, but is very useful when prototyping and sharing jsbins.
+
+Usage:
+
+    <head>
+      <base href="https://polygit.org/components/"> <!-- saves typing! -->
+      <script src="webcomponentsjs/webcomponents-lite.min.js"></script>
+      <link rel="import" href="paper-button/paper-button.html">
+      <link rel="import" href="iron-selector/iron-selector.html">
+    </head>
+
+For more documentation, see [http://polygit.org](http://polygit.org/).
+
+Source: [github.com/PolymerLabs/polygit](https://github.com/PolymerLabs/polygit)
+
+## Development tools {#dev}
+
+### <b>&lt;seed-element></b> - boilerplate starter for creating a resuable element on Github {#seedelement}
+
+See [Create a reusable element](/1.0/docs/start/reusableelements.html).
+
+### <b>Polymer Starter Kit</b> - starter project for building Polymer apps {#psk}
+
+The Polymer Starter Kit is a complete starting point for building Polymer-based applications See [Using Polymer Starter Kit](/1.0/docs/start/psk/set-up.html) for more information on getting started.
+
+Source: [github.com/PolymerElements/polymer-starter-kit](https://github.com/PolymerElements/polymer-starter-kit)
+
+### <b>polylint</b> - lint projects for errors and common mistakes {#polylint}
+
+[polylint](https://github.com/PolymerLabs/polylint) is a linting tool to detect
+common mistakes and errors in your projects.
+
+Install:
+
+    npm install -g polylint
+
+Usage:
+
+    polylint --root my-project/ \
+        --input elements.html my-element.html
+
+**Source:** [github.com/PolymerLabs/polylint](https://github.com/PolymerLabs/polylint)
+
+**Related tools**
+
+- [Sublime plugin](https://github.com/nomego/SublimeLinter-contrib-polylint)
+- [Atom package](https://github.com/PolymerLabs/polylint#installing-the-atom-package)
+
+### <b>polystyle</b> - web service for creating style modules {#polystyle}
+
+[polystyle](https://poly-style.appspot.com/demo/) is a web service that can wrap an existing stylesheet on a remote server as Polymer [style module](https://www.polymer-project.org/1.0/docs/devguide/styling.html#style-modules). This is useful if you have a hosted third-party stylesheet that you want to use in an element or your application.
```

Comment:
everyone keeps asking for a gulp version of this. Should we link to the MakleSoft one (or however you spell it :)